.swiftlint.yml

```diff
@@ -20,6 +20,10 @@ opt_in_rules:
 included:
   - Sources
   - Tests
+excluded: 
+   - Sources/Tron/Protobuf/Tron.pb.swift
+   - Sources/Tron/Protobuf/Discover.pb.swift
+   - Sources/Tron/Protobuf/Contract.pb.swift
```

Comment:
We can exclude the folder with `Sources/Tron/Protobuf`

Reply:
done